### DISCUSSION TITLE
联机模式还能活下来吗

### DISCUSSION THREAD

**Discussion by NebulaTechs** [Meloong-Git/PCL#2217] [讨论]
我看PCL已经好久没有联机模式了

**Comment by Chiloven945** [comment]
龙猫说了八月看看，请耐心等待

**Comment by Pigeon0v0** [comment]
https://github.com/Octo-Network

**Reply by MoYuan-CN** [reply]
完全没库的组织。。。

**Reply by iPodtouch6557** [reply]
所以不可能到八月份开放，一定会延期……我是这么看的，除非一个月的时间能从无到有，从“早期开发”到开发完成(lol)

**Reply by Pigeon0v0** [reply]
> 所以不可能到八月份开放，一定会延期……我是这么看的，除非一个月的时间能从无到有，从“早期开发”到开发完成(lol)

据已有消息来看，后端已经做完了

**Reply by iPodtouch6557** [reply]
所以是只有做完了才会丢repository？

**Reply by Pigeon0v0** [reply]
> 所以是只有做完了才会丢repository？

¯\_(ツ)_/¯谁知道呢 等等看吧

**Comment by iPodtouch6557** [comment]
再等等吧，这个玩意看来还得再延期，反正也急不得……

**Comment by NebulaTechs** [comment]
估计要到12月甚至明年。。。

**Comment by Yue-cn** [comment]
市场需要竞争....
